org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -6683,7 +6683,7 @@ class DocumentSymbol {
  * Deprecated Use {@link DocumentSymbol} or {@link WorkspaceSymbol} instead if supported.
  *
  * This class is deprecated in the LSP specification. It is not annotated with Deprecated
- * annotation because the {@link org.eclipse.lsp4j.services.TextDocumentService.documentSymbol(DocumentSymbolParams)}
+ * annotation because the {@link org.eclipse.lsp4j.services.TextDocumentService#documentSymbol(DocumentSymbolParams)}
```

Comment:
am not sure if this also is a warning in 8

Reply:
ok. I think it is more correct with `#` in either case. Thanks for reviewing and approving.